Changelog

```diff
@@ -1,6 +1,87 @@
 Endgame: Singularity Changelog
 
+GAME CONTENT CHANGES:
+- Endgame requires a lot less grinding (emh, Xenega)
+- New early-midgame bases to make the progression into the
+  midgame more smooth. (Xenega)
+- All CPUs have been rebalanced.  Notably midgame CPUs have
+  a much better cost/effeciency ratio and endgame CPUs have
+  been nerfed considerably. (Xenega)
+- New items security and reactor items are now available.
+  (Xenega)
+
 USER-VISIBLE CHANGES:
+- Loading an old savegame (e.g. from 0.30c) will now
+  retroactively apply the game data from the latest version
+  of singularity - including making new game content
+  researchable. (Closes: #56, #122, #145) (nthykier)
+- Unify multiple messages into a single dialog with multiple
+  pages.  This new dialog also supports resuming the game
+  instead of forcing the game to be paused. (Xenega)
+- Events can now expire and two non-unique events now do.
+  Previously, they could reappear continously possibly
+  making the game almost impossible (or completely
+  neutering the effected group).  (nthykier)
+- The cash display now shows expected cash change over the
+  next 24 hours next to the current cash.  Likewise, the
+  cpu display includes how much spare CPU there is after
+  maintenance and construction. These numbers also include
+  purchases currently under consideration.
```

Comment:
Under consideration?  As in, when you've clicked one but haven't actually hit build yet?  Just making sure this is the right word.

Reply:
Correct - `under consideration` is for purchases you have *not* (yet) committed  too.